src/views/ActivityProfile/activity-profile.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../vars';
+
+.gordon-activity-profile {
+  .activity-image {
```

Comment:
With larger activity images, the image goes beyond the width of the card.
With a longer list with names of different lengths, the email icon 
![screen shot 2017-12-16 at 12 58 20](https://user-images.githubusercontent.com/5354752/34073159-5aafd7d2-e261-11e7-91e4-273f59475f57.png)
We can fix that by limiting the width of the image so it fits in its container.
```CSS
.activity-image {
  margin-top: 25px;
  margin-bottom: 25px;

  img {
    max-width: 100%;
  }  
}
```